test/aerospike_clj/client_test.clj

```diff
@@ -64,18 +64,73 @@
     (is (= data @(client/get-single-no-meta *c* K _set)))))
 
 (deftest put-get-clj-map
-  (let [data {:foo {:bar [(rand-int 1000)]}}]
+  (let [data {"foo" {"bar" [(rand-int 1000)]}}]
     (is (true? @(client/create *c* K _set data 100)))
     (testing "clojure maps can be serialized as-is")
     (let [v @(client/get-single-no-meta *c* K _set)]
       (is (= data v)) ;; per value it is identical
-      (is (= java.util.HashMap (type v))) ;; but we get back a HashMap
-      (is (false? (map? v)))
-      (testing "using Jackson to recursively create a PersistantHashMap"
-        (let [json (.writeValueAsString (ObjectMapper.) v)
-              parsed (json/parse-string json #(keyword (subs % 1)))]
-          (is (= data parsed))
-          (is (true? (map? parsed))))))))
+      (is (= clojure.lang.PersistentArrayMap (type v))))))
+
+(deftest put-multiple-bins-get-clj-map
+  (let [data {"foo" {"bar" [(rand-int 1000)]}
+              "baz" true
+              "qux" false
+              "quuz" nil}]
+    (is (true? @(client/create *c* K _set data 100)))
+    (testing "clojure maps can be serialized from bins")
+    (let [v @(client/get-single-no-meta *c* K _set)]
```

Comment:
`let` should be inside `testing` :)

Reply:
Done. I really should be paying more attention to this